bin/setup

```diff
@@ -14,13 +14,16 @@ bin/start --no-dev
 [ $? != 0 ] && echo "Failed to start Docker services" && exit
 
 bin/clinotty chmod u+x bin/magento
-rm -rf src/magento-site && mkdir src/magento-site
+rm -rf src/magento-site && mkdir -p src/magento-site
 
 echo "Adding Magento modules to Composer allow-plugins directive..."
 bin/clinotty composer config --no-plugins allow-plugins.magento/magento-composer-installer true
 bin/clinotty composer config --no-plugins allow-plugins.magento/inventory-composer-installer true
 bin/clinotty composer config --no-plugins allow-plugins.laminas/laminas-dependency-plugin true
 
+echo "Uninstalling Magento..."
+bin/clinotty bin/magento setup:uninstall -y
```

Comment:
when I executed this, it says `The "-y" option does not exist.`

Reply:
from the documentation, looks like you need the "-n" option.
`abrahambosch@Abrahams-MacBook-Pro publicsquare-magento-payments-plugin % bin/clinotty bin/magento setup:uninstall --help
Description:
  Uninstalls the Magento application

Usage:
  setup:uninstall [options]

Options:
      --magento-init-params=MAGENTO-INIT-PARAMS  Add to any command to customize Magento initialization parameters
                                                 For example: "MAGE_MODE=developer&MAGE_DIRS[base][path]=/var/www/example.com&MAGE_DIRS[cache][path]=/var/tmp/cache"
  -h, --help                                     Display help for the given command. When no command is given display help for the list command
  -q, --quiet                                    Do not output any message
  -V, --version                                  Display this application version
      --ansi|--no-ansi                           Force (or disable --no-ansi) ANSI output
  -n, --no-interaction                           Do not ask any interactive question
  -v|vv|vvv, --verbose                           Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
`

Reply:
fixed.